pageserve/templates/index.html

```diff
@@ -40,9 +60,13 @@ <h2>Past posts</h2>
 <p>{{post.text}}</p>
 {% endif %}
 {% if post.files %}
-  {% for file in post.files %}
-  <img src="{{ file }}">
-  {% endfor %}
+{% for file in post.files %}
```

Comment:
Why flatten this indentation?

Reply:
Oh, that was unintentional. My autoformatter did that because all the HTML elements inside this jinja are siblings so this is technically correct. Should I unflatten?

Reply:
I think it's much easier to read when unflattened - is there a way to disable the flattening?

Reply:
I just disabled my autoformatter for HTML files and reindented the file manually https://github.com/knative-portability/large-events/pull/56/commits/6568c90c905771f17b84959aa39f9a457eb4a955